zerver/lib/actions.py

```diff
@@ -333,6 +368,7 @@ def do_set_realm_name(realm, name):
     return {}
 
 def do_set_realm_restricted_to_domain(realm, restricted):
+    # type: (Realm, bool) -> Dict[None, None]
```

Comment:
I think this group of functions should probably just not return anything -- nothing looks at those return values